src/cmd.js

```diff
@@ -82,7 +84,7 @@ function handleParallelList(
       }
 
       return possiblePromise.then(result => {
-        return Promise.all(result.map(a => dispatch(a)));
+        return Promise.all(result.map(a => wrappedDispatch(a)));
```

Comment:
the key is here. We use the wrapped dispatch rather than the vanilla one.

I could see something like this maybe being needed in the new timeout/delay Cmd types also, but we can deal with that after this is merged (or I will add it if that gets merged first)

Reply:
Yes, I also realized this will be an issue for Cmd.setTimeout. And yes we can solve it separately.